screeps-game-api/src/constants.rs

```diff
@@ -390,39 +394,53 @@ js_deserializable!(Terrain);
 /// implementations only operate on made-up integer constants. If you're ever
 /// using these impls manually, use the `__look_num_to_str` and
 /// `__look_str_to_num` JavaScript functions to convert.
+///
+/// To convert from a string representation directly, use
```

Comment:
As mentioned elsewhere, maybe specify that the best method depends on where the value lives?
Actually, it might be beneficial (in another PR) to add a brief explanation of the memory model in the crate's doc. It would avoid some confusion and explain design decisions. Thoughts?

Reply:
Adding documentation on the memory model in general sounds like a good idea!

For this particular PR, I've mentioned the two different contexts where one could use the two different conversion methods in the longer documentation for the module.